content/en/docs/reference/glossary/ephemeral-container.md

```diff
@@ -0,0 +1,18 @@
+---
+title: Ephemeral Container
+id: ephemeral-container
+date: 2019-08-26
+full_link: /docs/concepts/workloads/pods/ephemeral-containers/
+short_description: >
+  A container type that a user requests to run temporarily in a pod.
+
+aka:
+tags:
+- fundamental
+---
+ A container type that a user requests to run temporarily in a pod.
```

Comment:
@verb, is it more important to state that the ephemeral container is `temporarily run` or that the container is a debugging tool?
Is it correct to say, A type of container vs. A container type?
Could use a tooltip for container, {{< glossary_tooltip term_id="container" >}}